sentry-android-core/src/main/java/io/sentry/android/core/ScreenshotEventProcessor.java

```diff
@@ -69,60 +63,28 @@ public ScreenshotEventProcessor(
 
       return event;
     }
+    if (CurrentActivityHolder.getInstance().getActivity() == null
```

Comment:
```suggestion
    final Activity activity = CurrentActivityHolder.getInstance().getActivity()
    if (activity == null
```

M: Now you can reuse the `activity` object which lives only in the scope of this method.